sklearn/linear_model/_coordinate_descent.py

```diff
@@ -1536,12 +1538,21 @@ def fit(self, X, y, sample_weight=None):
             MSE that is finally used to find the best model is the unweighted
             mean over the (weighted) MSEs of each test fold.
 
+        **params : dict, default=None
+            Parameters to be passed to the CV splitter.
+
+            .. versionadded:: 1.4
+                Only available if `enable_metadata_routing=True`. See
```

Comment:
As in the other PR, I would suggest to add a reference to `set_config` and `config_context` to be explicit where `enable_metadata_routing` needs to be set.

Reply:
This would need to be done in all places.

Reply:
we can do this in a separate PR.